modules/nf-core/pyrodigal/meta.yml

```
@@ -33,10 +36,10 @@ output:
       type: file
       description: File containing software versions
       pattern: "versions.yml"
-  - gff:
+  - annotations:
       type: file
-      description: gene annotations in gff format
-      pattern: "*.{gff.gz}"
+      description: gene annotations in output format given as input
```

```suggestion
      description: gene annotations. The file format is specified via input channel "output_format".
```
Just updated the wording, I was confused by "gene annotations in output format given as input"